packages/qunit-dom/lib/install.ts

```diff
@@ -1,6 +1,12 @@
 import DOMAssertions from './assertions';
 import { getRootElement } from './root-element';
 
+declare global {
```

Comment:
I wanted to move this to a common location that was imported by both the IIFE and ES entrypoints.

Additionally, given the file name, install, I thought it makes semantic sense to live here as well, since we're installing the `dom` property on `Assert`